database-mongoose/index.js

```diff
@@ -15,83 +15,83 @@ conn.once('open', function () {
-//   cutomerCountry: String,
-//   profilePicUrl: String,
-//   title: String,
-//   description: String,
-//   rating: String,
-//   isVerifiedPurchase: Boolean,
-//   isHelpfulCount: Number,//updated
-//   imageUrlId: Array,
-//   reviewDate: Date,
-//   easeToUse: Number,
-//   voiceRecognistion: Number,
-//   techSupport: Number,
-//   valueForMoney: Number,
-//   qualityOfMaterial: Number,
-//   batteryLife: Number
+let ReviewsSchema = mongoose.Schema({
+  reviewId: Number,
+  productId: Number,
+  color: String,
+  configuration: String,
+  isBestSeller: Boolean,
+  category: String,
+  customerId: Number,
+  customerName: String,
+  cutomerCountry: String,
+  profilePicUrl: String,
+  title: String,
+  description: String,
+  rating: String,
+  isVerifiedPurchase: Boolean,
+  isHelpfulCount: Number,//updated
+  imageUrlId: Array,
+  reviewDate: Date,
+  easeToUse: Number,
+  voiceRecognistion: Number,
+  techSupport: Number,
+  valueForMoney: Number,
+  qualityOfMaterial: Number,
+  batteryLife: Number
 
+});
+
+const ReviewsModel = mongoose.model('Reviews', ReviewsSchema);
+//delete model to remove data
+// ReviewsModel.remove({}, function (err) {
+//   console.log('collection removed')
 // });
 
-// const ReviewsModel = mongoose.model('Reviews', ReviewsSchema);
-// //delete model to remove data
-// // ReviewsModel.remove({}, function (err) {
-// //   console.log('collection removed')
-// // });
-
-// let insertSeedData = (arrayOfObjects) => {
-//   arrayOfObjects.forEach(element => {
-//     let reviewDocument = new ReviewsModel({
-//       reviewId: element.reviewId,
-//       productId: element.productId,
-//       color: element.color,
-//       configuration: element.configuration,
-//       isBestSeller: element.isBestSeller,
-//       category: element.category,
-//       customerId: element.customerId,
-//       customerName: element.customerName,
-//       cutomerCountry: element.cutomerCountry,
-//       profilePicUrl: element.profilePicUrl,
-//       title: element.title,
-//       description: element.description,
-//       rating: element.rating,
-//       isVerifiedPurchase: element.isVerifiedPurchase,
-//       isHelpfulCount: element.isHelpfulCount,
-//       imageUrlId: element.imageUrlId,
-//       reviewDate: element.reviewDate,
-//       easeToUse: element.easeToUse,
-//       voiceRecognition: element.voiceRecognition,
-//       techSupport: element.techSupport,
-//       valueForMoney: element.valueForMoney,
-//       qualityOfMaterial: element.qualityOfMaterial,
-//       batteryLife: element.batteryLife
-
-//     });
-//     reviewDocument.save(function (err, doc) {
-//       if (err) {
-//         console.log(err);
-//         return err;
-//       }
-//       console.log('Document inserted: ', element.reviewId, element.configuration);
-//       return ('insertSeedData');
-
-//     });
-
-//   });
-
-
-// }
-
-// module.exports.insertSeedData = insertSeedData;
+let insertSeedData = (arrayOfObjects) => {
```

Comment:
Consider moving insertSeedData function into own file or with other seeder functionality (separation of concerns).

Reply:
Since data is an arrayOfObjects and you've already formatted each object based on the schema, could db.collection.insertMany() be used here?

Reply:
Regarding 'Consider moving insertSeedData function into own file' , index.js is my db model where DB connection and   all CRUD operations will be done here. Shouldn't all db related operations happen here?

Regarding 'db.collection.insertMany()' - Agree. Made note of it.

Reply:
Your reasoning for keeping insertSeedData() inside db file makes sense as well. I do not see a problem with leaving as is 🙂.